### ISSUE TITLE
code style and ml

### ISSUE THREAD

**Issue by zkud** [widbnudb/MyClothes#1]
**Ревью** 
https://github.com/widbnudb/MyClothes/blob/master/Code/MyClothes_gui.py
* Не хватает пробельных строк вначале
* Импорты по pep 8 должны идти в следующем порядке:
 1. стандартные библиотеки
 2. импорты сторонних библиотек
 3. импорты модулей текущего проекта
* super(MenuWindow, self).__init__(parent) можно заменить на super().__init__(parent)  
----
https://github.com/widbnudb/MyClothes/blob/master/Code/Recognizer/ImageHandler.py  
* Нужно исправить отступы & положение фигурных скобок [пример (чуть полиствать вниз до my_list & result) ](https://pythonworld.ru/osnovy/pep-8-rukovodstvo-po-napisaniyu-koda-na-python.html#id3)  
* Нужно добавить разделение пробелами частей кода в  методах imageprepare, recognizer
----
https://github.com/widbnudb/MyClothes/blob/master/Code/Recognizer/MyClothes_neural.py
* Пробельные строки
* Импорты (как в вышеописанных модулях)  
* Лишний импорт Dropout
* Терминология & моменты, связанные с машинным обучением  

`# Добавляем уровни сети  
model.add(Dense(900, input_dim=784, activation="relu"))  
model.add(Dense(10, activation="softmax"))` 

1. Вернее будет не "уровни", а слои
2. Почему нельзя взять уже обученную модель + дообучить под задачу (экономия ресурсов + обучение быстрее)
3. Почему бы не добавить визуализаций обучения (чтобы пользоваться нормально early stopping, видеть, переобучается ли модель и тд)

`model.compile(loss="categorical_crossentropy",
              optimizer="SGD",
              metrics=["accuracy"])`
1. У SGD есть проблемы с элипсообразными поверхностями, часто 'шумит' на них. Был ли попробован Adam?
2. Может быть стоит добавить что - нибудь к метрикам, например, precision, recall, f1 и тп.

 **Дополнительные предложения** 
* Использовать ORM для работы с БД
* Добавить __pycache__ в .gitignore

**Крутой проект! Получил удовольствие от ревью!**
